library/network_connections.py

```diff
@@ -1554,23 +1554,34 @@ def _complete_kwargs_loglines(self, rr, connections, idx):
             )
             prefix = prefix + (", '%s'" % (c["name"]))
         for r in rr["log"]:
```

Comment:
It would be great if you could change this to something like

`for loglevel, b, c in rr["log"]` with good names for loglevel, b, c. Then these names could be used instead of `r[0]` etc in the next line to make it easier to understand. Hope you still know what these fields are.

Reply:
I've change this for a better readability, thanks for the tip!